src/components/breadcrumb/components/breadcrumb-item/breadcrumb-item.stories.js

```diff
@@ -0,0 +1,35 @@
+import { html } from 'lit-html';
+import { docsIntro, wrapWidth, TYPES } from '../../../../../.storybook/utils.js';
+import '../..';
+
+export default {
+  title: 'custom-elements/vl-breadcrumb/vl-breadcrumb-item',
+  parameters: {
+    controls: { hideNoControlsWarning: true },
+    docs: {
+      description: {
+        component: docsIntro({
+          root: 'breadcrumb',
+          stylesheets: ['breadcrumb'],
+          intro: 'Use breadcrumb items to add extra navigation steps to a breadcrumb.',
+        }),
+      },
+    },
+  },
+  args: {
+    href: '',
+  },
+  argTypes: {
+    href: {
+      name: 'data-vl-href',
+      description: 'Changes the url to which the browser will navigate when clicking the breadcrumb item.',
+      type: { summary: TYPES.STRING },
+      defaultValue: { summary: '#' },
```

Comment:
Die `#` is niet echt de default value van dat attribuut van de component. Default wilt zeggen dat als het niet wordt gezet, in de component zelf toch die waarde wordt gebruikt. De `defaultValue` prop mag dus gewoon weg hier.